src/ops/direnv/mod.rs

```diff
@@ -15,8 +17,24 @@ pub fn main(project: &Project) -> OpResult {
     let mut shell_root = project.gc_root_path().unwrap();
     shell_root.push("build-0"); // !!!
 
+    // TODO: don’t start build/evaluation automatically, let the user decide
```

Comment:
I don't understand the TODO, a ping should be always desirable, no?

Reply:
No, most of the time I’m developing I don’t want to start any kind of build in the background, I just want it to put me in the environment of the last build. Though it’s course that’s debatably the whole point of lorri that the developer doesn’t have to think about it.

Reply:
Also probably best to discuss in an issue and then decide there.

Reply:
issue is at: https://github.com/target/lorri/issues/100